steps/cli/oc.go

```diff
@@ -165,6 +167,18 @@ var _ = gauge.Step("Update addon config with resolverStepActions as <resolverSte
 	}
 })
 
+var _ = gauge.Step("Verify versioned ecosystem tasks", func() {
+	taskList := cmd.MustSucceed("oc", "get", "task", "-n", "openshift-pipelines").Stdout()
+	requiredTasks := []string{"buildah", "git-cli", "git-clone", "kn", "kn-apply", "maven", "openshift-client", "s2i-dotnet", "s2i-go", "s2i-java", "s2i-nodejs", "s2i-perl", "s2i-php", "s2i-python", "s2i-ruby", "skopeo-copy", "tkn"}
+	operator.VerifyVersionedTasks(taskList, requiredTasks)
+})
+
+var _ = gauge.Step("Verify versioned ecosystem step actions", func() {
+	stepActionList := cmd.MustSucceed("oc", "get", "stepaction", "-n", "openshift-pipelines").Stdout()
```

Comment:
Why don't you move these two lines into the function? Does it have any advantages to keep it here?

Reply:
Done